api/package.json

```diff
@@ -1,31 +1,40 @@
 {
-  "name": "@freecodecamp/api",
-  "version": "0.0.1",
+  "author": "freeCodeCamp <team@freecodecamp.org>",
+  "bugs": {
+    "url": "https://github.com/freeCodeCamp/freeCodeCamp/issues"
+  },
+  "dependencies": {
+    "@fastify/middie": "8.0.0",
+    "@fastify/mongodb": "6.1.0",
+    "fastify": "4.9.2",
+    "fastify-plugin": "4.3.0"
+  },
   "description": "The freeCodeCamp.org open-source codebase and curriculum",
-  "license": "BSD-3-Clause",
-  "private": true,
   "engines": {
     "node": ">=18",
     "npm": ">=8"
   },
+  "homepage": "https://github.com/freeCodeCamp/freeCodeCamp#readme",
+  "license": "BSD-3-Clause",
+  "main": "none",
+  "name": "@freecodecamp/api",
+  "nodemonConfig": {
+    "env": {
+      "NODE_ENV": "development"
+    },
+    "ignore": [
+      "**/*.js"
+    ]
+  },
+  "private": true,
   "repository": {
     "type": "git",
     "url": "git+https://github.com/freeCodeCamp/freeCodeCamp.git"
   },
-  "bugs": {
-    "url": "https://github.com/freeCodeCamp/freeCodeCamp/issues"
-  },
-  "homepage": "https://github.com/freeCodeCamp/freeCodeCamp#readme",
-  "author": "freeCodeCamp <team@freecodecamp.org>",
-  "main": "none",
   "scripts": {
     "build": "tsc index.ts",
     "start": "NODE_ENV=production node index.js",
-    "develop": "NODE_ENV=development nodemon index.ts"
+    "develop": "nodemon index.ts"
```

Comment:
Want to try the new watch mode from Node 18?

Reply:
I did not even know that was a feature 🤩 . I like the sound of that, but we will still need to use `ts-node`.

Reply:
Ah – I keep forgetting that we are on TypeScript.